src/commands/utils/github/functions/handleGithubIssue.ts

```diff
@@ -0,0 +1,44 @@
+import { ChatInputCommandInteraction, Colors } from 'discord.js';
+import { request } from '@octokit/request';
+import { makeEmbed } from '../../../../lib';
+
+const syntaxHelp = '\nSyntax:\nA32NX repo: `/github issue <id>`\nAny FBW repo: `/github issue <repo> <id>`';
+
+const noQueryEmbed = makeEmbed({
+    title: 'Issue Error | Missing Query',
+    description: `Invalid command!${syntaxHelp}`,
+    color: Colors.Red,
+});
+
+const invalidEmbed = makeEmbed({
+    title: 'Issue Error | Invalid',
+    description: `Something went wrong! Did you provide the correct repo/Issue id?${syntaxHelp}`,
+    color: Colors.Red,
+});
+export async function handleGithubIssue(interaction: ChatInputCommandInteraction<'cached'>) {
+    const issueNumber = interaction.options.getString('issue_number');
+    const repoName = interaction.options.getString('repo');
+
+    if (!issueNumber) return interaction.reply({ embeds: [noQueryEmbed] });
+
+    const cleanedIssueNumber = issueNumber.replace('#', '');
+
+    if (repoName) {
+        try {
+            const response = await request('GET /repos/flybywiresim/{repo}/issues/{pull_number}', {
+                repo: repoName,
+                pull_number: cleanedIssueNumber,
```

Comment:
```suggestion
            const response = await request('GET /repos/flybywiresim/{repo}/issues/{issue_number}', {
                repo: repoName,
                issue_number: cleanedIssueNumber,
```
Just a bit cleaner in context

Reply:
Totally missed these ones in my dumb copy paste! Thanks